.github/CONTRIBUTING.md

```diff
@@ -37,6 +37,15 @@ Some hints:
  * Models sent/received by the controllers are defined in [/shared/models](/shared/models) directory
 
 
+## Improve the website
+
+PeerTube's website is [joinpeertube.org](https://joinpeertube.org), where people can learn about the project and how it works – note that it is not a PeerTube instance, but more the “marketing website”.
+
+You can help us improve it too!
+
+It is not hosted on GitHub but rather on [Framasoft](https://framasoft.org/)'s own [GitLab](https://about.gitlab.com/) instance, [FramaGit](https://framagit.org): https://framagit.org/framasoft/peertube/joinpeertube
```

Comment:
That doesn't tell what to make of that information: bug reports, code review… Until the repository gets a CONTRIBUTING guide of some sort, we should quickly explain that.

Reply:
Right, there's not much info about how to contribute in the FramaGit repo. But then wouldn't it be better to get said repo a proper README with contributing instructions?

Reply:
Yes :slightly_smiling_face: -> https://framagit.org/framasoft/peertube/joinpeertube/issues/10

Reply:
Good call! But then we should take care of that in joinpeertube.org's repo and leave this section like that, right? 🙂